helper_posix_test.go

```diff
@@ -133,6 +124,10 @@ func copyTestdataFile(path string) [][]string {
 	return copyTestdataFileTo(path, path)
 }
 
+func rawCopyTestdataFile(path string) string {
+	return rawCopyTestdataFileTo(path, path)
+}
+
```

Comment:
Note, you don't need to create this method. The test that uses it fails validation _before_ it runs the task, so there is no need for the task to actually copy artifacts. Since it fails validation before running, it doesn't matter if the file has been copied or not.